reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,322 @@
+
+For more information about Modules, see
+[about_Modules](/powershell/module/Microsoft.PowerShell.Core/About/about_Modules).
+
+### Separate profiles
+
+A PowerShell profile is a script that executes when PowerShell starts. This script customizes
+your environment by adding commands, aliases, functions, variables, modules, and PowerShell drives.
+The profile script makes these customizations available in every session without having to manually
+recreate them.
+
+The path to the location of the profile has changed in PowerShell 7.
+
+- In Windows PowerShell 5.1, the location of the profile is `$HOME\Documents\WindowsPowerShell`.
+- In PowerShell 7, the location of the profile is `$HOME\Documents\PowerShell`.
+
+The profile filenames have also changed:
+
+   ```powershell
+   PS> $PROFILE | Select-Object *Host* | Format-List
+
+   AllUsersAllHosts       : C:\Program Files\PowerShell\7\profile.ps1
+   AllUsersCurrentHost    : C:\Program Files\PowerShell\7\Microsoft.PowerShell_profile.ps1
+   CurrentUserAllHosts    : C:\Users\<user>\Documents\PowerShell\profile.ps1
+   CurrentUserCurrentHost : C:\Users\<user>\Documents\PowerShell\Microsoft.PowerShell_profile.ps1
+   ```
+
+For more information [about_Profiles](/powershell/module/microsoft.powershell.core/about/about_profiles).
+
+### PowerShell 7 compatibility with Windows PowerShell 5.1 modules
+
+Most of the modules you use in Windows PowerShell 5.1 already work with PowerShell 7, including
+Azure PowerShell and Active Directory. We're continuing to work with other teams to add native
+PowerShell 7 support for more modules including Microsoft Graph, Office 365, and others. For the
+current list of supported modules, see
+[PowerShell 7 module compatibility](/powershell/scripting/whats-new/module-compatibility).
+
+[!NOTE]
+> On Windows, we've also added a **UseWindowsPowerShell** switch to `Import-Module` to ease the
+> transition to PowerShell 7 for those using incompatible modules. For more information on this
+> functionality, see
+> [about_Windows_PowerShell_Compatibility](/powershell/modules/Microsoft.PowerShell.Core/About/about_windows_powershell_compatibility).
+
+### PowerShell Remoting
+
+PowerShell remoting lets you run any PowerShell command on one or more remote computers. You can
+establish persistent connections, start interactive sessions, and run scripts on remote computers.
+
+#### WS-Management remoting
+
+Windows PowerShell 5.1 and below use the `WS-Management` protocol for connection negotiation and
+data transport. If remoting has been enabled, installing PowerShell 7 will use the existing
+**WinRM** configuration.
+
+> [!NOTE]
+> PowerShell 7 will use the existing Windows PowerShell 5.1 endpoint for remoting
+> connections. To update PowerShell 7 to include it own endpoint, run the
+> `Install-PowerShellRemoting.ps1` located in `$PSHOME`. For information about connecting to
```

Comment:
Recommend folks to use `Enable-PSRemoting` from PS7 instead of running this script.  Also should include an example of remoting to a PS7 endpoint.

Reply:
Fixed